docs/mastering-plone/installation.md

```diff
@@ -151,9 +151,8 @@ Follow the instructions of `docs.plone.org`: {doc}`plone6docs:volto/development/
 
 ### Option 2. Start with prepared training project `volto-ploneconf` with all code for the training
 
-Prepare the pre-requisites explained in {ref}`plone6docs:install-packages-prerequisites-label`.
+Prepare the pre-requisites explained in {ref}`plone6docs:create-project-cookieplone-prerequisites-for-installation-label`.
```

Comment:
We've removed the hyphen from prerequisites in the main docs, as it is a more common spelling. I'll merge this commit, and merge the PR when green. Thank you!

```suggestion
Prepare the prerequisites explained in {ref}`plone6docs:create-project-cookieplone-prerequisites-for-installation-label`.
```